README.md

```diff
@@ -71,13 +71,17 @@ Default file type can choose by `markdown.marp.exportType` preference.
 
 > ⚠️ Export except HTML requires to install [Google Chrome](https://www.google.com/chrome/) (or [Chromium](https://www.chromium.org/)). You may also specify the custom path for Chrome and Chromium-based browser by preference `markdown.marp.chromePath` (e.g. [Microsoft Edge](https://www.microsoft.com/edge)).
 
+### Accessing User's `settings.json`
+
+On the command pallet of VS Code, type Open Settings (JSON).
+
 ### Use custom theme
 
 You can register and use [custom theme CSS for Marpit](https://marpit.marp.app/theme-css) / [Marp Core](https://github.com/marp-team/marp-core/tree/master/themes#readme) by setting `markdown.marp.themes`, that includes remote URLs, or relative paths to local files in the current workspace.
 
 ```javascript
-// Please put `.vscode/settings.json` on your workspace
 {
+  // Add this entry to your User's `settings.json`
```

Comment:
It works but we don't recommend setting `markdown.marp.themes` in user settings because it may have a relative path from workspace.

It may cause performance degradation of VS Code by unnecessary file I/O when settings this in user's `settings.json`.

Reply:
I confess I didn't understand what you mean by that.
>  it may have a relative path from workspace
Wouldn't the original path be based on the user's home folder, therefore also a relative path, regarding the user?


> It may cause performance degradation of VS Code by unnecessary file I/O when settings this in user's settings.json.

I thought we were only adding a key in a dictionary. I couldn't see at all how having one extra value would harm it. Which show I don't understand the details on the implementation of VS Code.

Reply:
OK, let me make clear that  `.vscode/settings.json` is based on current VS Code workspace, not user's home directory.

The setting value of theme CSS path explained in this context allows URL, and relative path from current workspace root. By security reason, we cannot load local CSS file from the outside of current workspace (#110, https://github.com/microsoft/vscode/issues/45260). So setting value of relative path should be unique by workspace.

VS Code has 2 different scopes for settings: User Settings and Workspace Settings (https://code.visualstudio.com/docs/getstarted/settings). Theme CSS may be workspace-specific setting by above reason, so workspace settings is better than user settings. If there is theme CSS path in user settings, Marp for VS Code will try to find out CSS from disk even if opening the unrelated workspace so may lead performance degression.